RoboticonColony/Assets/Code/Classes/Inventory.cs

```diff
@@ -0,0 +1,103 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+/// <summary>
+/// An inventory. Holds all resources for a player or market and facilitates transactions with other inventories.
+/// </summary>
+sealed public class Inventory
```

Comment:
Can't see why not, will have to wait for Tom to get finished with his Tile class though. The inventory could just be the place where player owned tiles are stored anyway couldn't it?